### PR TITLE
Add a FAQ section to the API docs

### PR DESCRIPTION
**Pull request by holiber**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -257,3 +257,189 @@ To disable that behavior, use the `compactMode` parameter in JSON-RPC request:
...
+# FAQ
+
+## What is the difference between Sources and SceneItems?
+
+One thing to keep in mind in OBS is that sources and scene items are different things.
+A source exists outside the context of a scene. Muted is a property of a source. 
+A scene item is essentially an instance of a source in a scene. 
+The same underlying source can be in multiple scenes as a sceneitem. 
+Visibility is an attribute on a scene item. 
+So the same source can be visible in one scene and invisible in another for example.
```
Comment (andycreeth) [inline docs/README.md]: This looks familiar 😆

File: docs/README.md [hunk 2]
```diff
@@ -257,3 +257,189 @@ To disable that behavior, use the `compactMode` parameter in JSON-RPC request:
...
+            "muted": false,
+            "name": "AuxAudioDevice1",
+            "type": "wasapi_input_capture"
+            "id": "wasapi_input_capture_d497319b"
+        }
+    ]
+}
+```
+
+Usually the AudioSource with name `AuxAudioDevice1` is mike.
```
Comment (andycreeth) [inline docs/README.md]: Usually the AudioSource with the name `AuxAudioDevice1` is the microphone.

File: docs/README.md [hunk 3]
```diff
@@ -257,3 +257,189 @@ To disable that behavior, use the `compactMode` parameter in JSON-RPC request:
...
+        "x": 975.2116452561818,
+        "y": 392.0672268907563,
+        "scaleX": 1,
+        "scaleY": 1,
+        "visible": true
+    }
+}
+```
+
+Now you have a `resourceId` for `ISceneItemApi` instance and can use it's methods includes `setVisibility`.
```
Comment (andycreeth) [inline docs/README.md]: Now you have a `resourceId` for the `ISceneItemApi` instance and can use its methods including `setVisibility`.

File: docs/README.md (hunk 1 above)
Comment (holiber) [reply docs/README.md]: Yeah, that is the coarse theft of my laziness =)

**Pull request merged**